packages/shared/src/utils/browser/browser.js

```diff
@@ -0,0 +1,25 @@
+export const parseUA = user_agent => {
+    const lookup = [
```

Comment:
Just my two cents, but in the future we should think of using an external library for browser lookup? Since browser sniffing via `user_agent` isn't reliable in the long run as browsers tend to change their names in user agent lists and it would be a pain to maintain this ourselves. 

Maybe worth looking at something like `bowser`?

Reply:
Good point @zaki-binary 
I took a glance at `browser` code. It does the job exactly the same way we do but with a greater support for browsers, different OSs and mobile devices.
I prefer to use it in our project :+1: